geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/ingest/IngestCommand.scala

```diff
@@ -31,12 +32,14 @@ trait IngestCommand[DS <: DataStore] extends DataStoreCommand[DS] {
   def libjarsPaths: Iterator[() => Seq[File]]
 
   override def execute(): Unit = {
-    import DataFormats.{Avro, Csv, Shp, Tsv}
+    import DataFormats.{Avro, Csv, Shp, Tsv, Json}
 
     ensureSameFs(PathUtils.RemotePrefixes)
 
     val ingest = if (params.fmt == Shp) {
       createShpIngest()
+    } else if (params.spec == null && params.config == null && params.fmt == Json) {
```

Comment:
would it make sense to have geojson separate from json as an input format?

Reply:
I considered that. Right now it's kind of hidden that they're separate. Non-geo json would ingest normally is the converter is specified, but I can define a geojson input format if that would make it more clear in the code.

Do we infer the format based on the extension? Because I've seen geojson files ending in json, though I guess it's not too much to ask the user to rename.

Reply:
I'm guess I'm just wondering what happens if you pass in a normal json file without any converter or anything - it's probably ok to keep them the same, as long as we handle that case with a decent error message